config/config-reporting/connector.go

```diff
@@ -0,0 +1,56 @@
+package config_reporting
+
+import (
+	"encoding/json"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/rudderlabs/rudder-server/config"
+	"github.com/rudderlabs/rudder-server/utils/httpconnector"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+)
+
+var (
+	pkgLogger        logger.LoggerI
+	configBackendURL string
+)
+
+func Init() {
+	pkgLogger = logger.NewLogger().Child("config-reporting")
+	loadConfig()
+}
+
+func loadConfig() {
+	configBackendURL = config.GetEnv("CONFIG_BACKEND_URL", "https://api.rudderlabs.com")
+}
+
+// ReportDataPlaneConfig sends the data-plane information to the control-plane.
+func ReportDataPlaneConfig() error {
+	planeConfig := dataPlaneConfig{
+		Namespace: os.Getenv("NAMESPACE"),
```

Comment:
Even when we run it on our local this would translate to an empty string.Let's have a default value over here?

```suggestion
		Namespace: config.Getenv("NAMESPACE","non-k8s-deployment"),
```

Reply:
Make sense